src/components/header/Header.jsx

```diff
@@ -28,22 +31,25 @@ const Header = () => {
             <nav className="nav-action-btns">
                 <ul className="horizontal-align-centre nav-text-primary">
                     <li className="list-style-none">
-                        <button>
-                            <Link to="#" className="btn btn-secondary btn-sm border-squre">
-                                Login
-                            </Link>
+                        <button onClick={() => {
+                            navigate("/signup")
+
+                        }}>
```

Comment:
Instead of programmatically navigating on the click of a button, you can wrap the button inside a Link tag

Reply:
thanks for pointing out